docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  mysql:
+    image: mysql:5.7
+    restart: on-failure
+    environment:
+      - MYSQL_ROOT_PASSWORD=ci_root_pw
+      - MYSQL_HOST=course_inventory_mysql
+      - MYSQL_PORT=3306
+      - MYSQL_DATABASE=course_inventory_local
+      - MYSQL_USER=ci_user
+      - MYSQL_PASSWORD=ci_pw
```

Comment:
I would suggest adding these mysql credentials default values to `env_blank.json` sample as well

Reply:
I'm gonna hold off on this for now;  not everyone may be using docker-compose. Instruction are provided in the README. Requirements are also changing, we may not shift to using Oracle.